app/coffee/HistoryManager.coffee

```diff
@@ -11,14 +11,19 @@ module.exports = HistoryManager =
 		if !Settings.apis?.trackchanges?
 			logger.warn { doc_id }, "track changes API is not configured, so not flushing"
 			return
-
-		url = "#{Settings.apis.trackchanges.url}/project/#{project_id}/doc/#{doc_id}/flush"
-		logger.log { project_id, doc_id, url }, "flushing doc in track changes api"
-		request.post url, (error, res, body)->
-			if error?
-				logger.error { error, doc_id, project_id}, "track changes doc to track changes api"
-			else if res.statusCode < 200 and res.statusCode >= 300
-				logger.error { doc_id, project_id }, "track changes api returned a failure status code: #{res.statusCode}"
+		RedisManager.getHistoryType doc_id, (err, projectHistoryType) ->
+			if err?
+				logger.error {err, doc_id}, "error getting history type"
+			else if projectHistoryType is "project-history"
+				logger.debug {doc_id, projectHistoryType}, "skipping track-changes flush"
+			else
+				url = "#{Settings.apis.trackchanges.url}/project/#{project_id}/doc/#{doc_id}/flush"
+				logger.log { project_id, doc_id, url, projectHistoryType }, "flushing doc in track changes api"
```

Comment:
should we downgrade this to debug?

Reply:
Maybe in a separate PR, it was log originally so I'll leave it at that for now.